src/includes/getting-started-primer/flutter.mdx

```diff
@@ -12,15 +12,18 @@ Features:
 - Offline storage of events
 - Events [enriched](/platforms/flutter/enriching-events/context/) with device data
 - Breadcrumbs automatically captured:
-    -  by the [Flutter SDK](/platforms/flutter/enriching-events/breadcrumbs/#automatic-breadcrumbs)
+    - by the [Dart SDK](/platforms/dart/enriching-events/breadcrumbs/#automatic-breadcrumbs)
+    - by the [Flutter SDK](/platforms/flutter/enriching-events/breadcrumbs/#automatic-breadcrumbs)
     - via the Native SDKs [Automatic Breadcrumbs for Android](/platforms/android/enriching-events/breadcrumbs/#automatic-breadcrumbs) and [Automatic Breadcrumbs for iOS](/platforms/apple/configuration/integrations/default/#sentryautobreadcrumbtrackingintegration)
-    - as well as `http` with the [Dart SDK](/platforms/dart/)
+    - by the [Logging Integration](/platforms/dart/configuration/integrations/logging)
+    - by the [Dio Integration](/platforms/dart/configuration/integrations/dio/)
 - [Release Health](/product/releases/health/) tracks crash free users and sessions
 - [Attachments](/platforms/flutter/enriching-events/attachments/) enrich your event by storing additional files, such as config or log files.
 - [User Feedback](/platforms/flutter/enriching-events/user-feedback/) provides the ability to collect user information when an event occurs.
 - [Performance Monitoring](/product/performance/) creates transactions for:
     - [HTTP requests](/platforms/flutter/performance/instrumentation/automatic-instrumentation/#httpclient-library-instrumentation)
     - [Routing Instumentation](/platforms/flutter/performance/instrumentation/automatic-instrumentation/#routing-instumentation)
+    - [Dio Integration](/platforms/dart/configuration/integrations/dio/#performance-monitoring-for-http-requests)
```

Comment:
What about call it Dio HTTP library instead of "Integration"?

```suggestion
    - [Dio HTTP library](/platforms/dart/configuration/integrations/dio/#performance-monitoring-for-http-requests)
```